tests/e2e/default/translations/nepali-dates-and-numbers.wdio-spec.js

```diff
@@ -129,16 +110,18 @@ describe('Bikram Sambat date display', () => {
     const date1 = '2021-01-01';
     const dateBk1 = bikramSambat.toBik_text(date1);
 
-    await reportsPage.setDateInput('/bikram-sambat-dates/data/date1', date1);
-    expect(await reportsPage.getFieldValue('/bikram-sambat-dates/data/date1_text')).to.equal(dateBk1);
-    expect(await reportsPage.getSummaryField('/bikram-sambat-dates/summary/field1')).to.equal(`date1 = ${dateBk1}`);
+    await commonEnketoPage.setDateValue('Date 1', date1);
+    await genericForm.formTitle().click();
+    expect(await commonEnketoPage.getInputValue('to-bikram-sambat([date 1 path])')).to.equal(dateBk1);
 
-    const date2 = '2021-01-01';
+    const date2 = '2021-02-02';
```

Comment:
I changed it because now we are using the text value to locate the element, and to see that both dates are reflected in the summary of the form, they need to be different